### DISCUSSION TITLE
Is there a way to hide logs by default?

### DISCUSSION THREAD

**Discussion by ethanabrooks** [fonsp/Pluto.jl#2015] [Questions about using Pluto] [answered]
The latest version of Pluto prints out terminal logs by default.

On the one hand, the ability visualize terminal logs is very useful.

On the other hand, my logs are very, very long and I use Pluto for presentations. Consequently, I have to go through and click "Hide logs" every time I reload the page:

![image](https://user-images.githubusercontent.com/10344742/160862642-85f950bd-00f1-44d6-a3a1-650731b066d3.png)

Is there a way to have logs start hidden?

**Comment by Pangoraw** [comment, marked as answer]
Hi! There are currently no way to hide logs persistently. But the ability to persist hiding logs should be coming soon (See https://github.com/fonsp/Pluto.jl/pull/2023)!

**Comment by ethanabrooks** [comment]
I think I actually solved this: in `HTTP.request` I had set `verbose=3`. Setting this to `0` suppressed log output.

**Reply by Pangoraw** [reply]
That's also a great solution when you have access to the function logging!
